tests/test_ext_apidoc.py

```diff
@@ -188,3 +188,80 @@ def test_extension_parsed(make_app, apidoc):
     with open(outdir / 'conf.py') as f:
         rst = f.read()
         assert "sphinx.ext.mathjax" in rst
+
+
+@pytest.mark.apidoc(
+    coderoot='test-apidoc-toc',
+    options=["--implicit-namespaces"],
+)
+def test_toc_all_references_should_exist_pep420_enabled(make_app, apidoc):
+    """All references in toc should exist. This test doesn't say if
+       directories with empty __init__.py and and nothing else should be
+       skipped, just ensures consistency between what's referenced in the toc
+       and what is created. This is the variant with pep420 enabled.
+    """
+    outdir = apidoc.outdir
+    assert (outdir / 'conf.py').isfile()
+
+    toc = extract_toc(outdir / 'mypackage.rst')
```

Comment:
It seems the behavior of `sphinx-apidoc` has changed between stable and master.
```
# master
$ sphinx-apidoc -o zzz -F tests/roots/test-apidoc-toc/ --implicit-namespaces
Creating file zzz/test-apidoc-toc.mypackage.rst.
Creating file zzz/test-apidoc-toc.mypackage.no_init.rst.
Creating file zzz/test-apidoc-toc.mypackage.something.rst.
Creating file zzz/conf.py.
Creating file zzz/index.rst.
Creating file zzz/Makefile.
Creating file zzz/make.bat.
```
```
# stable
$ sphinx-apidoc -o zzz -F tests/roots/test-apidoc-toc/ --implicit-namespaces
Creating file zzz/mypackage.rst.
Creating file zzz/mypackage.no_init.rst.
Creating file zzz/mypackage.something.rst.
Creating file zzz/conf.py.
Creating file zzz/index.rst.
Creating file zzz/Makefile.
Creating file zzz/make.bat.
```

Reply:
I found this change comes from #3962 at master.
So this testcase should be written.